docker-compose.yml

```diff
@@ -1,25 +1,5 @@
 version: '2'
 services:
-  app:
```

Comment:
I would still keep an `app` service, so that we keep the consistency we have in all apps with `dc run app`